app/renderer/components/task_card.js

```diff
@@ -39,17 +41,18 @@ module.exports = {
             <div class="col s1">
                 <progress-spinner v-if="running && AppStatus.config.animatedSpinner"></progress-spinner>
                 <i v-else class="small material-icons" v-bind:style="{color: statusColor}">{{task.status}}</i>
+                <tooltip v-bind:task="task"></tooltip>
```

Comment:
This is better, but probably binding task.status will make the code in Tooltip easier and less coupled with Task